tests/app/test_model.py

```diff
@@ -161,6 +162,20 @@ def test_query_live_brief(self):
 
             assert Brief.query.filter(Brief.status == 'draft').count() == 0
             assert Brief.query.filter(Brief.status == 'live').count() == 1
+            assert Brief.query.filter(Brief.status == 'withdrawn').count() == 0
```

Comment:
Would be nice to test in a few places whether the python implementation and `expression` implementation of `status` always agree (these lines only test the `expression` implementation).